Show nodes and handles from component

ollimeier opened the pull request:
Fixes #1630

This is how it looks like:

<img width="1547" alt="Screenshot 2024-09-12 at 09 01 23" src="https://github.com/user-attachments/assets/4e016d26-1d2b-4452-9c16-afc99cc8f191">
<img width="1547" alt="Screenshot 2024-09-12 at 09 01 26" src="https://github.com/user-attachments/assets/cb75e1d0-b36e-49d3-b83d-c928c3af7a75">

justvanrossum commented:
Thanks!

Looking at the screenshot, I'm not sure about the stroke treatment: it causes these nodes/handles to be _more_ visible somehow. I would love the nodes to be drawn solid, but quite dimmed, so that they are _just_ visible, but clearly different from the editable nodes/handles.

ollimeier commented:
> Thanks!
> 
> Looking at the screenshot, I'm not sure about the stroke treatment: it causes these nodes/handles to be _more_ visible somehow. I would love the nodes to be drawn solid, but quite dimmed, so that they are _just_ visible, but clearly different from the editable nodes/handles.

What about this:
<img width="1547" alt="Screenshot 2024-09-12 at 10 29 42" src="https://github.com/user-attachments/assets/01c71740-04b0-44f3-b096-0dc361997243">
<img width="1547" alt="Screenshot 2024-09-12 at 10 29 46" src="https://github.com/user-attachments/assets/3bebea99-0364-42f9-9dd5-3ab5853b1030">

justvanrossum commented:
If you're worried about how visible they are on top of a zone: maybe we should use opacity.

ollimeier commented:
> If you're worried about how visible they are on top of a zone: maybe we should use opacity.

Thanks for your feedback. I added a minor opacity, added a language key and made some adjustments to the zIndex. 
What do you think?

justvanrossum reviewed (approved):
Done! Feel free to merge.
